spec/dor/rights_metadata_spec.rb

```diff
@@ -50,19 +50,73 @@
     before :each do
       allow(OpenURI).to receive(:open_uri).with('https://purl-test.stanford.edu/bb046xn0881.xml').and_return('<xml/>')
     end
+
     it 'should have correct primary' do
       expect(Dor.logger).to receive(:warn).with(/Cannot index druid:bb046xn0881\.descMetadata/)
       doc = @item.to_solr
+
       expect(doc).to match a_hash_including(
-        'rights_primary_ssi'  => 'world_qualified',
-        'metadata_source_ssi' => 'DOR',
-        'title_tesim'         => ['Indianapolis 500'],
```

Comment:
`rights_primary_ssi`, `metadata_source_ssi`, `title_tesim` should be looking for the same field values only the formatting of the hash has changed w/ regards to these fields.